Delete the example workflows since they are already included in the README

stefafafan opened the pull request:
<!--
Thank you for proposing a pull request! Please note that SOME TESTS WILL
LIKELY FAIL due to how GitHub exposes secrets in Pull Requests from forks.
Someone from the team will review your Pull Request and respond.

Please describe your change and any implementation details below.
-->

## Description
The example GitHub workflows in this repository use `Authenticating via Service Account Key JSON` .
Using Service Account Key JSON values here is deprecated, as shown here:
- https://github.com/google-github-actions/deploy-cloudrun#inputs
- https://github.com/google-github-actions/setup-gcloud#authentication-inputs

When someone new wants to try out this github action and sees the example, they would naturally start using something that is deprecated.

## What I've done
Given this comment https://github.com/google-github-actions/deploy-cloudrun/pull/343#issuecomment-1124197925, I deleted the example workflow files so the user can just use the examples in the README.

sethvargo commented:
Hi @stefafafan 

I actually think we should just delete these files, as the examples are already in the README.

stefafafan commented:
@sethvargo Thank you for the review.
You have a point, I went ahead and deleted the example workflows (and also renamed the Pull Request, updated the description). How is this?

irab commented:
@sethvargo You might want to revert this PR, as the README.md currently links to these workflows and the there is not example in there for a example workflow that includes a build step - explicitly mentioned here: https://github.com/marketplace/actions/deploy-to-cloud-run#build-and-deploy-a-container
